docs/enterprise/install/quickstart.md

```diff
@@ -129,12 +129,21 @@ Open your Pomerium config file, `/etc/pomerium/config.yaml`.
          pass_identity_headers: true
    ```
 
-1. If you haven't already, set `signing_key`. See the [reference page](/reference/readme.md#signing-key) for more information.
+   The example value for `to:` assumes Pomerium and Pomerium Enterprise are running on the same test environment.
+
+1. If you haven't already, set `signing_key`. See the [reference page](/reference/readme.md#signing-key) for more information. The same signing key must be used in both Pomerium Core and Enterprise.
 
    ```yaml
    signing_key: "ZZZZZZZZZZZZZZ"
    ```
 
+1. Define the databroker storage type and connection string. The example below assumes a local Redis server:
```

Comment:
Maybe link to the databroker storage docs here?

Reply:
https://github.com/pomerium/pomerium/pull/2480/commits/b15463590b5af5a8073dd752e2fffef894a48bcf